voluncheer/opportunityboard/test_forms.py

```diff
@@ -54,7 +54,7 @@ def test_validation(self):
             "description": "Let's surfing",
             "staffing": "1",
             "date": timezone.now(),
```

Comment:
I see this isn't in this PR but I would be remise to leave it alone. Generally you should never use a time.now() like function in a test, it makes the test non-deterministic. Rather you should create a "fake" date and time such that it is always the same.

Reply:
The `setUp() `test cases for opportunity board were a mess, so I created a helper function in unittest_setup.py that could be imported to make the test opportunities more consistent. Now all tests avoid using `timezone()` to create the start datetime.